frontend/src/metabase/App.tsx

```diff
@@ -10,7 +10,7 @@ import {
   NotFound,
   Unauthorized,
 } from "metabase/containers/ErrorPages";
-import UndoListing from "metabase/containers/UndoListing";
+import { UndoListing } from "metabase/containers/UndoListing";
```

Comment:
Start review here

Reply:
Updated `UndoListing` to be a named export, since they are preferred over default exports. See our [FE style guide notion doc.](https://www.notion.so/metabase/FE-Style-Guide-Proposals-Discussion-acdddeb36bd148e592d8be749ff9e4ef?pvs=4#01ed8c871d224d0daf5c45ee05e6b46e)

Reply:
Actually, I'm not sure that's the case, most of Metabase exports are default 🤔 I think it was a proposal we haven't really implemented (cc @alxnddr)

Reply:
That's true, the vast majority of engineers preferred using default exports. I'll drop this section from the doc

Reply:
FWIW I think it's worth at least keeping in the doc even if many engineers don't follow and we don't enforce it strictly. Perhaps in the future we can enforce it with a [lint rule](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-default-export.md).